app/src/main/java/org/p2p/wallet/rpc/repository/signature/RpcSignatureRepository.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.wallet.rpc.repository.signature
+
+import org.p2p.solanaj.core.PublicKey
+import org.p2p.solanaj.model.types.SignatureInformation
+
+interface RpcSignatureRepository {
+    suspend fun getConfirmedSignaturesForAddress(
```

Comment:
We need one more function here:
It was about getting transaction details.
I guess, it's `getConfirmedTransaction`.

Thus, these two functions are being used in history screen, so let' call it HistoryRepository or smth like that